specifyweb/frontend/js_src/lib/components/FormEditor/Editor.tsx

```diff
@@ -195,8 +196,10 @@ function UseLabelsSchema(): JSX.Element {
     globalThis.location.reload();
```

Comment:
can you confirm that this the unload protect was correctly set, but wasn't being triggered by this `reload()` command?

i.e if you make changes and then try to use the back/forward browser buttons, do you get the unload protect dialog?

Reply:
no you don't